src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```
@@ -3,27 +3,44 @@
 $spacing: 15px;
 $close-icon-width: 20px;
 $message-margin: $spacing + $close-icon-width + 5px;
+$action-button-height: 30px;
+$action-button-padding: 6px;
 
-.announcement-container {
+.AnnouncementMessage {
   display: block;
-  z-index: z(announcement);
-  position: absolute;
+  position: relative;
   background: $primary;
-  top: 66px;
-  left: 50%;
-  width: 500px;
-  margin: 0 auto 0 -250px;
+  width: 638px;
   transition: left 0.1s linear;
 
-  &.centered {
-    left: 50%;
-  }
-
   padding: $spacing 0;
-  border-radius: 0 0 20px 20px;
+  border-radius: 20px;
```

```suggestion
  border-radius: $border-radius--xl;
```
While they are not the same, the difference is not significant

---

https://github.com/sparkletown/sparkle/pull/1440/commits/2508808ee3450f7dcd481b08e7a60183dacc3a4f